frontends/concrete-python/tests/compilation/test_program.py

```diff
@@ -69,13 +252,21 @@ def dec(x):
         p_error=0.1,
         parameter_selection_strategy="v0",
         composable=True,
-        verbose=True,
     )
-    Module.compile(
+    artifacts = fhe.ModuleDebugArtifacts(["inc", "dec"])
```

Comment:
Maybe make function names optional and them being unspecified mean all functions?

Reply:
Done